### PR TITLE
BRIGHT_RED colored error trace output

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Colored error tracing.

Ready for review.

### CODE REVIEW

File: libimagutil/src/trace.rs [hunk 1]
```diff
@@ -29,9 +33,11 @@ pub fn trace_error(e: &Error) {
 /// Output is the same as for `trace_error()`, though there are only `max` levels printed.
 pub fn trace_error_maxdepth(e: &Error, max: u64) {
     let n = count_error_causes(e);
-    write!(stderr(), "{}/{} Levels of errors will be printed", (if max > n { n } else { max }), n);
-    print_trace_maxdepth(n, e, max);
-    write!(stderr(), "");
+    let mut term = stderr().unwrap();
```
Comment (TheNeikos) [inline libimagutil/src/trace.rs]: When could this panic?

Comment (matthiasbeyer) [inline libimagutil/src/trace.rs]: After reading through https://stebalien.github.io/doc/term/src/term/lib.rs.html#106-108 and https://github.com/Stebalien/term/blob/fe676a3abe189de841c174ef53c7d35b8afdddb0/src/terminfo/mod.rs#L303-L305 it seems that this only panics if there cannot be a terminal opened. At least there is no `panic!()` in the crates root afaics. Shall I add a comment on this?

Comment (TheNeikos) [inline libimagutil/src/trace.rs]: So if I pipe the output into a file this might panic?

Comment (matthiasbeyer) [inline libimagutil/src/trace.rs]: I'm not sure. I will test as soon as I can.

Comment (matthiasbeyer) [inline libimagutil/src/trace.rs]: Jop, it panics if `$TERM` is not set. I will implement this in another way.

Comment (matthiasbeyer) [inline libimagutil/src/trace.rs]: Waiting on https://github.com/Stebalien/term/issues/57

**Pull request closed without merging**